sdk/ios/start.md

```diff
@@ -180,13 +178,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 ```
 
-Next, in your application code where you wish to use the AppSync client (like your View Controller) reference this in the `viewDidLoad()` lifecycle method:
+Next, in your application code where you wish to use the AppSync client (like your View Controller) reference this in the `viewDidLoad()` lifecycle method or in the instantiation of your view when using SwiftUI views.
```

Comment:
This sentence has gotten unwieldily. Can we split it into two? Maybe this can be reworded as:

> Next, reference the AppSync client that you initialized in the AppDelegate, above. You could do so inside of your `viewDidLoad()` lifecycle method, or while instantiating a SwitfUI view. See the examples below.

Reply:
thanks, updating